EnglishFilter/sections/content_blocker.txt

```diff
@@ -1,6 +1,11 @@
 !
-! Section contains list of the rules for content blockers
-! It will be used by AdGuard Content Blocker and AdGuard for iOS/Safari
+! This section contains the list of the rules for content blockers (AdGuard for iOS, AdGuard for Safari and AdGuard Content Blocker for Yandex Browser / Samsung Browser on Android).
```

Comment:
```suggestion
! This section contains the list of the rules for content blockers (AdGuard for iOS, AdGuard for Safari and AdGuard Content Blocker for Yandex Browser / Samsung Browser on Android).
!
! The point is that these apps support only a limited subset of AdGuard filtering rules syntax. 
! Because of that, we have to resort to using more complicated and less effective solutions there.
```